analyzers/tests/SonarAnalyzer.UnitTest/Rules/InsecureEncryptionAlgorithmTest.cs

```diff
@@ -26,27 +28,52 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class InsecureEncryptionAlgorithmTest
     {
+        private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.InsecureEncryptionAlgorithm>();
+        private readonly VerifierBuilder builderVB = new VerifierBuilder<VB.InsecureEncryptionAlgorithm>();
+
         [TestMethod]
         public void InsecureEncryptionAlgorithm_MainProject_CS() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\InsecureEncryptionAlgorithm.cs", new CS.InsecureEncryptionAlgorithm(), GetAdditionalReferences());
+            builderCS.AddPaths("InsecureEncryptionAlgorithm.cs")
+                .AddReferences(GetAdditionalReferences())
+                .Verify();
 
         [TestMethod]
         public void InsecureEncryptionAlgorithm_DoesNotRaiseIssuesForTestProject_CS() =>
-            OldVerifier.VerifyNoIssueReportedInTest(@"TestCases\InsecureEncryptionAlgorithm.cs", new CS.InsecureEncryptionAlgorithm(), GetAdditionalReferences());
+            builderCS.AddPaths("InsecureEncryptionAlgorithm.cs")
+                .AddTestReference()
+                .AddReferences(GetAdditionalReferences())
+                .VerifyNoIssueReported();
 
 #if NET
         [TestMethod]
```

Comment:
NITPICK:
According to our coding style: Each compiler directive outside method body (namely #if/#endif) should be preceded and followed by an empty line.
Which means you need an empty line both there and and the endif.